release-notes/unreleased/2520-Show-number-of-search-results.yml

```
@@ -0,0 +1,6 @@
+issue_key: 2520
+show_in_stores: false
```

```suggestion
show_in_stores: true
```

Please set this to true, because this is a new feature we want this information in the stores.